atomic_reactor/inner.py

```diff
@@ -336,10 +336,8 @@ class ImageBuildWorkflowData(ISerializer):
     # post-build plugins (such as koji_import) to record them.
     all_yum_repourls: List[str] = field(default_factory=list)
 
-    # Plugins can store info here using the @annotation, @annotation_map,
-    # @label and @label_map decorators from atomic_reactor.metadata
+    # Plugins can store info here using the @annotation, @annotation_map
```

Comment:
The comment looks a bit incomplete (maybe just remove "the"?)

Reply:
updated